README.md

```diff
@@ -1,14 +1,47 @@
 # astrojs-with-umbraco-cms-docs
 Drafts of docs to present to AstroJS - https://docs.astro.build/en/guides/cms/
 
-## Umbraco CMS
-
 [Umbraco CMS](https://umbraco.com/) is an open-source ASP.NET Core CMS.
 
-## Prerequisites
+Umbraco serves as both a CMS with a built-in frontend and a headless CMS via its Content Delivery API that can be used to provide content to your Astro project.
+
+## Integrating with Astro
+
+Umbraco offers a built-in Content Delivery API for connecting your Umbraco content to any application.
+
+You can also utilise [Umbraco Heartcore](https://umbraco.com/products/umbraco-heartcore/), a hosted headless version of Umbraco.
```

Comment:
If we mention Heartcore, we may need to explain any differences in setup. Could potentially leave it off for the first pass to avoid complicating things.

Reply:
Yeah I think we should omit Heartcore in this instance - I personally have no way to test it out, and it isn't technically "Umbraco CMS".

Reply:
Yeah, agreed. Keep it simple to start with